docs/deploy-with-olm.md

```diff
@@ -109,6 +109,37 @@ the Block CR, you may run in to issues where the pod reports being unschedulable
 
 In this example, the disks were prevsiously used, and a filesystem was applied.  Be sure to either use fresh disks, or remove any partitions and file systems if you're setting up a Block mode CR.
 
+### Create a CR using Tolerations
+
+The following example assumes that some nodes in the cluster have [taints](https://docs.openshift.com/container-platform/latest/nodes/scheduling/nodes-scheduler-taints-tolerations.html) that designate them for use with the local-storage Operator.
+
+For instance, assume that some nodes have a key, `localstorage`, with a value of `true`. With the following configuration the local-storage Operator will only search for volumes on nodes that match this key, or nodes that are generally available for scheduling:
+
```

Comment:
This documentation of taints is not entirely accurate. Tolerations don't behave like a selector and this makes it sound like one.

Reply:
Updated this section to be more explicit that this will match the specified taints, along with generally available nodes.

Reply:
How about a simple - "In addition to node selector, you can also specify tolerations for pods created by local-storage-operator, so as if selected nodes are tainted local-storage-operator pods can still run on those nodes" ?

Reply:
This text has been updated to largely incorporate your statement. I still have two paragraphs, the second leading into the example itself.